Nodejs/Product/Nodejs/Commands/DiagnosticsCommand.cs

```diff
@@ -285,8 +286,6 @@ private class FileTypeInfo {
                     jsAnalyzer.DumpLog(writer);
                 }
             }
-
-            res.AppendLine(string.Format("IntelliSense Completion Only Tab or Enter to Commit: {0}", NodejsPackage.Instance.IntellisenseOptionsPage.OnlyTabOrEnterToCommit));
```

Comment:
There's a bunch of other logic about the analyzer in this file that we should also remove unless it's also logging the IW window (or there's an easy way to make it do so).